modules/repository/create_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"testing"
 
 	"code.gitea.io/gitea/models"
```

Comment:
Seems as if we still haven't refactored enough…
I wonder when that'll be the case?